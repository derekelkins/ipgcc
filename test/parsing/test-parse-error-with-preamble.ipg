Junk

%preamble_end

S -> for i = 1 to 10 do A;
