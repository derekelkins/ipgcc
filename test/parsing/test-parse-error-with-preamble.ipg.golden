S -> for i = 1 to 10 do A;
                         ^
Parse error at line 5, column 26.
Possible following tokens are: '['